packages/wonder-blocks-button/src/themes/default.ts

```diff
@@ -339,12 +339,12 @@ const theme = {
     font: {
         size: {
             // NOTE: This token is specific to this button size.
-            large: 18,
+            large: "1.8rem",
         },
         lineHeight: {
-            small: tokens.font.lineHeight.small + textUnderlineOffset,
-            default: tokens.font.lineHeight.medium + textUnderlineOffset,
-            large: tokens.font.lineHeight.medium + 2 + textUnderlineOffset,
+            small: tokens.font.lineHeight.xMedium,
+            default: tokens.font.lineHeight.large,
+            large: tokens.font.lineHeight.large,
```

Comment:
So this has taken some fiddling as you can't compute REM units by adding them together. I wonder if we can land on specific values instead of adding magic numbers together, since the `textUnderlineOffset` is constant. The extra `2` in `font.lineHeight.large` is very much a one-off, so I opted to remove it.

One alternative idea would be to create a REM addition function that removes the `rem` from the units, adds them together, and puts `rem` back on the end. Or we could create a one-off rem value in this file?

Reply:
I thin one-off (including comment) makes total sense. I had to do something similar in a separate PR for the same reason.

Reply:
I added a one-off unit in there!